app/ui/labels.py

```diff
@@ -257,7 +262,12 @@ def on_delete_click(self, name, is_dynamic):
             )
             with snowpark_session(self.snowflake) as txn:
                 # Make the old label, bypassing validation
-                label_to_del = ModelLabel.construct(name=name)
+                if is_dynamic:
+                    label_to_del = ModelLabel.construct(
+                        group_name=name, is_dynamic=True
```

Comment:
I'm confused :(   Isn't it supposed to be group_name=**group** here too ?

Reply:
The `name` variable here is either the `Label.name` or `Label.group_name`. It's just confusingly named.

https://github.com/sundeck-io/OpsCenter/pull/342/files/d4f1dc2d1c80c03e6a53c52b52aa732bb5c34c6f#diff-ca2f4a6b94e630f89baef30ac659472d5a12a08d453cdca6ae60e64ded9339d9L136-L148

Reply:
Thank you so much for making the change, my sanity is restored :)